tests/test_5_mut.py

```diff
@@ -466,25 +464,34 @@ def test_sanitize():
             assert_equal(err.getvalue().strip(),
                          'Warning: while appending section: 2 to parent: 0\nThe section first point should be parent section last point: \n        : X Y Z Diameter\nparent last point :[2.000000, 0.000000, 0.000000, 2.000000]\nchild first point :[2.000000, 1.000000, 0.000000, 2.000000]')
 
-
 def test_glia():
     g = GlialCell()
     assert_equal(g.cell_family, CellFamily.GLIA)
 
-    g = GlialCell(os.path.join(_path, 'astrocyte.h5'))
+    g = GlialCell(Path(DATA_DIR, 'astrocyte.h5'))
     assert_equal(g.cell_family, CellFamily.GLIA)
 
-    g = GlialCell(Path(_path, 'astrocyte.h5'))
+    g = GlialCell(Path(DATA_DIR, 'astrocyte.h5'))
     assert_equal(g.cell_family, CellFamily.GLIA)
 
-    assert_raises(RawDataError, GlialCell, Path(_path, 'simple.swc'))
-    assert_raises(RawDataError, GlialCell, Path(_path, 'h5/v1/simple.h5'))
+    assert_raises(RawDataError, GlialCell, Path(DATA_DIR, 'simple.swc'))
+    assert_raises(RawDataError, GlialCell, Path(DATA_DIR, 'h5/v1/simple.h5'))
 
 
 def test_glia_round_trip():
     with TemporaryDirectory() as folder:
-        g = GlialCell(os.path.join(_path, 'astrocyte.h5'))
+        g = GlialCell(Path(DATA_DIR, 'astrocyte.h5'))
         filename = Path(folder, 'glial-cell.h5')
         g.write(filename)
         g2 = GlialCell(filename)
         assert_equal(len(g.sections), len(g2.sections))
+
+def test_lifetime_issue():
+    m = Morphology(DATA_DIR / 'simple.swc')
+    s = m.root_sections[0]
+    del m # ~mut.Morphology() called
+    with assert_raises(RuntimeError) as obj:
+        s.children
+    assert_substring("This section (id: 0) can no longer be used in the context of the whole "
+                     "morphology because the Morphology object it belongs to has been deleted",
+                     str(obj.exception))
```

Comment:
can you add a few more tests here; ie: moving sections from morph to morph, deleting the section, deleting the morphs, etc, so it's obvious what is allowed and what isn't?